lib/src/events.dart

```diff
@@ -29,7 +29,7 @@ class SockJSCloseEvent extends SockJSEvent {
   final String reason;
 
   /// Whether or not the close was "clean".
-  final bool wasClean;
+  final bool? wasClean;
```

Comment:
I think we could make this non-nullable because the JS interop event that we use to populate this field is also always non-null. If we do that, I think we'd need to mark the `wasClean` constructor param as `required`.